src/functions/template-tags/general.php

```diff
@@ -215,7 +215,12 @@ function tribe_get_datetime_format( $with_year = false ) {
 	 * @return mixed|void
 	 */
 	function tribe_get_time_format( ) {
-		$format = get_option( 'time_format' );
+		static $format;
```

Comment:
```suggestion
		static $format = get_option( 'time_format' );
```

Reply:
If we set default static var here, we might not need the conditional below

Reply:
Switched to using `tribe_[get|set]_var()` so we can override the value in tests.